rapid_response_xblock/models.py

```diff
@@ -41,3 +41,22 @@ def __str__(self):
                 answer_id=self.answer_id,
             )
         )
+
+
+class RapidResponseBlockStatus(models.Model):
+    """
+    Indicates whether a rapid-response-enabled XBlock for a given course is "open" or not
+    ("open" = set to collect student responses for the block in real time)
+    """
+    usage_key = UsageKeyField(max_length=255, db_index=True)
+    course_key = CourseKeyField(max_length=255, db_index=True)
+    open = models.BooleanField(default=False, null=False)
+
+    def __unicode__(self):
```

Comment:
Can you change this to `__str__` and add the `@python_2_unicode_compatible` decorator similar to the class above?